tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/api/PostResource2Test.java

```diff
@@ -315,21 +314,6 @@ public void getPostPlacementShouldReturnUnauthWhenUserNotPartOfSameTrustNew() th
     verify(placementSummaryDecorator, never()).decorate(any());
   }
 
-  @Test
-  public void deletePostShouldReturnUnauthWhenUserNotPartOfSameTrust() throws Exception {
-    long personId = 1L;
-    postDTO.setId(personId);
-
-    doThrow(new AccessUnauthorisedException("")).when(postService)
-        .canLoggedInUserViewOrAmend(personId);
-
-    restPostMockMvc.perform(delete("/api/posts/{id}", personId)
-        .contentType(MediaType.APPLICATION_JSON))
-        .andExpect(status().isUnauthorized())
-        .andExpect(jsonPath("$.message").value("error.accessDenied"));
-    verify(postService, never()).delete(any());
-  }
```

Comment:
An equivalent test is still useful for the LocalOffice restrictions.
Please make sure you are adequately covering the ACs.  I've checked a few files for a test but don't see an existing one.